plugin/vite.config.datacatalog-web.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="vitest" />
 /// <reference types="vite/client" />
+/// <reference types="vite-plugin-svgr/client" />
```

Comment:
I don't think you need this (or the other references) inside the config files since you have the plugin/env.d.ts file with them